src/api/Feed.js

```diff
@@ -533,6 +533,7 @@ class Feed extends Base {
                 }
             },
             errorCallback: (e: ElementsXhrError, code: string) => {
+                this.updateFeedItem({ isPending: false }, task.id);
```

Comment:
Are we showing any visual indication that the update failed?

Reply:
Yes. TaskModal has an error prop which renders an inlineError.